CMakePCHCompiler.cmake

```diff
@@ -67,6 +67,27 @@ function(target_precompiled_header) # target [...] header
 			set(target_dir
 				${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/${pch_target}.dir
 			)
+		endif()
+		if(MSVC)
+			# ensure pdb goes to the same location, otherwise we get C2859
+			get_filename_component(
+				pdb_dir
+				"${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/${target}.dir"
+				ABSOLUTE
+				)
+			get_filename_component(win_pch "${target_dir}/${header}.pch" ABSOLUTE)
+			get_filename_component(win_header "${header}" ABSOLUTE)
+		else()
+			# HACK: This lets the user include the header in their project
```

Comment:
While I understand the change. This isn't really related for MSVC subject of the commit. Would be nice to have it in separate commit and leave out the `WTF` 😃

Reply:
For what it's worth in case like this I write WTF specifically for things I mean to remove in the final version. It's a marker like HACK that says "REMOVE ME" but I don't want to delete the code, yet. I don't know why I'm being so patient today. (Sometimes I change it to WTH if I don't want to remove it just yet. But I'm not so concerned for temporary commits. I don't care if it's in version history IOW. And I don't care about the language either, so much as its unprofessional tone.)